lib/prx_auth/rails/token.rb

```diff
@@ -0,0 +1,32 @@
+require 'rack/prx_auth'
+
+class PrxAuth::Rails::Token
+  def initialize(token_data)
+    @token_data = token_data
+    @namespace = PrxAuth::Rails.configuration.namesapce
```

Comment:
Typo?

Reply:
yes, clearly need some tests in this one